tests/v2_validation/cattlevalidationtest/core/test_webhook.py

```
@@ -0,0 +1,116 @@
+                                    launchConfig=launch_config)
+    service = client.wait_success(service)
+    assert service.state == "inactive"
+    service.activate()
+    service = client.wait_success(service)
+    assert service.state == "active"
+
+    # Create webhook to scale up service by 2
+    url = "http://localhost:8080/v1-webhooks/receivers?projectId="+env.accountId
+    data = {
+        "name": "newservice 1 up-2",
+        "driver": "scaleService",
+        "scaleServiceConfig": {
+            "action": "up",
+            "amount": 2,
+            "serviceId": service.id,
+            "min": 1,
+            "max": 4
+        }
+    }
+    headers = {"Content-Type": "application/json", "Accept": "application/json"}
+    # r = requests.post(url, auth=(env.access_key, env.secret_key), data=json.dumps(data), headers=headers)
+    r = requests.post(url, data=json.dumps(data), headers=headers)
+    r.json()
+    assert r.status_code == 200
+
+    if r.status_code != 200:
+        assert r.content is not None
+        return
+
+    assert r.url is not None
+    resp = json.loads(r.content)
+    webhookTokenUp = resp["url"]
+
+    # Create webhook to scale down service by 2
+    data = {
+        "name": "newservice 1 up-2",
+        "driver": "scaleService",
+        "scaleServiceConfig": {
+            "action": "down",
+            "amount": 2,
+            "serviceId": service.id,
+            "min": 1,
+            "max": 4
+        }
+    }
+    r = requests.post(url, data=json.dumps(data), headers=headers)
+    r.json()
+    assert r.status_code == 200
+
+    if r.status_code != 200:
+        assert r.content is not None
+        return
+
+    assert r.url is not None
+    resp = json.loads(r.content)
+    webhookTokenDown = resp["url"]
+
+    # Execute webhookTokenUp once so scale of service becomes 3
+    r = requests.post(webhookTokenUp)
+    service = client.wait_success(service)
+    assert r.status_code == 200
```

you should do this above the wait_success bc the response is not 200, you want to fail fast and not do the wait_success logic